src/wms/src/test/java/org/geoserver/wms/map/GetMapKvpRequestReaderTest.java

```diff
@@ -97,6 +94,13 @@ protected void oneTimeSetUp() throws Exception {
         gi2.getStyles().add(getCatalog().getStyleByName("raster"));
         cb.calculateLayerGroupBounds(gi2);
         getCatalog().add(gi2);
+
+        LayerGroupInfo gi3 = cf.createLayerGroup();
+        gi3.setName("testGroup3");
+        gi3.getLayers().add(null);
+        gi3.getStyles().add(getCatalog().getStyleByName(MockData.BASIC_STYLE_GROUP.getLocalPart()));
```

Comment:
Just use a regular string for the style name here, or define a constant in this test class